### ISSUE TITLE
[Request/graphic issue] Design Fix

### ISSUE THREAD

**Issue by TheSign01** [Necrobot-Private/NecroBot#1547]
As always, GREAT technical work, but after the latest patch, I feel like I need to point out those two "issues" with the Win GUI design.
- First, a couple of weeks ago I could check how many items and pokemons I had, right from the labels on the left side, but those were replaced with icons with no utility.
- Second, the thing that brought me here: The console background color change. Now it blends with the text color, and takes too much attention from the eye.
Those two things are maybe meant to make the bot more "user friendly" but imo makes it less functional.

**Comment by morbosc** [comment]
Console colors were changed to a palette specifically for readability and reduced eye strain.  There was a conversation during the commit about adjusting them slightly for more contrast.  In particular, I have noted that Recycling and Info events don't have enough contrast.  I was looking at the CLI version of the app while making those particular changes (and trying to approximate colors to their CLI equivalent), I might have chosen colors that were intended for the lighter variant's background in those cases.

I'll test some variants and see if I can address the issue without making my eyes bleed like the original console.  The reference work is here, if you're curious about the palette:  http://ethanschoonover.com/solarized

Currently, a lot of theming is going on in testing, and there is a chance that once that is merged, I'll immediately revisit this (it was my commit), to give back what I feel is a very eye-strain-inducing color palette, along with the options to go with dark and light versions of these.

Item and pokemon counts are also readded in builds that are currently being tested.

Thanks for the feedback!

**Comment by vsq13** [comment]
show!!!

**Comment by CDAGaming** [comment]
The HydraBuild PR contains these new counts and testing with them is at 98 percent completion

**Comment by Lord-Haji** [comment]
Please do look to the already submitted issues before you open a new one
Duplicate of #1499
